### ISSUE TITLE
root error

### ISSUE THREAD

**Issue by hic9507** [vincenzosantopietro/action-recognition-efficientnet#3]
When I execute main.py, I had an error

(aiml) C:\Users\user\.conda\envs\aiml> python main.py --batch_size 32 --epochs 10 --efficientnet_id 0
2022-03-23 16:05:20.075883: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cudart64_101.dll
Found 151500 files belonging to 101 classes.
Using 121200 files for training.
2022-03-23 16:05:26.202442: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library nvcuda.dll
2022-03-23 16:05:26.221992: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties:
pciBusID: 0000:01:00.0 name: NVIDIA GeForce RTX 3090 computeCapability: 8.6
coreClock: 1.845GHz coreCount: 82 deviceMemorySize: 24.00GiB deviceMemoryBandwidth: 871.81GiB/s
2022-03-23 16:05:26.222112: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cudart64_101.dll
2022-03-23 16:05:26.227163: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cublas64_10.dll
2022-03-23 16:05:26.230112: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cufft64_10.dll
2022-03-23 16:05:26.231148: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library curand64_10.dll
2022-03-23 16:05:26.234328: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cusolver64_10.dll
2022-03-23 16:05:26.236002: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cusparse64_10.dll
2022-03-23 16:05:26.242895: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cudnn64_7.dll
2022-03-23 16:05:26.243114: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0
2022-03-23 16:05:26.243838: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2022-03-23 16:05:26.250636: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x27a1cc65000 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2022-03-23 16:05:26.250714: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2022-03-23 16:05:26.251164: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties:
pciBusID: 0000:01:00.0 name: NVIDIA GeForce RTX 3090 computeCapability: 8.6
coreClock: 1.845GHz coreCount: 82 deviceMemorySize: 24.00GiB deviceMemoryBandwidth: 871.81GiB/s
2022-03-23 16:05:26.251411: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cudart64_101.dll
2022-03-23 16:05:26.251587: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cublas64_10.dll
2022-03-23 16:05:26.251800: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cufft64_10.dll
2022-03-23 16:05:26.252057: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library curand64_10.dll
2022-03-23 16:05:26.252328: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cusolver64_10.dll
2022-03-23 16:05:26.252510: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cusparse64_10.dll
2022-03-23 16:05:26.252679: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cudnn64_7.dll
2022-03-23 16:05:26.252915: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0
2022-03-23 16:08:20.530134: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2022-03-23 16:08:20.530262: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      0
2022-03-23 16:08:20.530482: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1276] 0:   N
2022-03-23 16:08:20.530688: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1402] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 21821 MB memory) -> physical GPU (device: 0, name: NVIDIA GeForce RTX 3090, pci bus id: 0000:01:00.0, compute capability: 8.6)
2022-03-23 16:08:20.532798: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x27a6b0cfc90 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:
2022-03-23 16:08:20.532837: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): NVIDIA GeForce RTX 3090, Compute Capability 8.6
Found 151500 files belonging to 101 classes.
Using 30300 files for validation.
Found 15150 files belonging to 101 classes.
Model: "functional_1"
_________________________________________________________________
Layer (type)                 Output Shape              Param #
=================================================================
input_1 (InputLayer)         [(None, 220, 220, 3)]     0
_________________________________________________________________
efficientnetb0 (Functional)  (None, 7, 7, 1280)        4049571
_________________________________________________________________
average_pooling2d (AveragePo (None, 3, 3, 1280)        0
_________________________________________________________________
flatten (Flatten)            (None, 11520)             0
_________________________________________________________________
dropout (Dropout)            (None, 11520)             0
_________________________________________________________________
dense (Dense)                (None, 101)               1163621
=================================================================
Total params: 5,213,192
Trainable params: 5,171,169
Non-trainable params: 42,023
_________________________________________________________________
2022-03-23 16:08:29.573810: I tensorflow/core/profiler/lib/profiler_session.cc:164] Profiler session started.
2022-03-23 16:08:29.573899: I tensorflow/core/profiler/internal/gpu/cupti_tracer.cc:1391] Profiler found 1 GPUs
2022-03-23 16:08:29.574931: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'cupti64_101.dll'; dlerror: cupti64_101.dll not found
2022-03-23 16:08:29.575616: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'cupti.dll'; dlerror: cupti.dll not found
2022-03-23 16:08:29.575668: E tensorflow/core/profiler/internal/gpu/cupti_tracer.cc:1441] function cupti_interface_->Subscribe( &subscriber_, (CUpti_CallbackFunc)ApiCallback, this)failed with error CUPTI could not be loaded or symbol could not be found.
Epoch 1/10
2022-03-23 16:08:36.835879: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cublas64_10.dll
2 root error(s) found.
  (0) Invalid argument:  assertion failed: [Unable to decode bytes as JPEG, PNG, GIF, or BMP]
         [[{{node decode_image/cond_jpeg/else/_1/decode_image/cond_jpeg/cond_png/else/_20/decode_image/cond_jpeg/cond_png/cond_gif/else/_39/decode_image/cond_jpeg/cond_png/cond_gif/Assert/Assert}}]]
         [[IteratorGetNext]]
         [[IteratorGetNext/_2]]
  (1) Invalid argument:  assertion failed: [Unable to decode bytes as JPEG, PNG, GIF, or BMP]
         [[{{node decode_image/cond_jpeg/else/_1/decode_image/cond_jpeg/cond_png/else/_20/decode_image/cond_jpeg/cond_png/cond_gif/else/_39/decode_image/cond_jpeg/cond_png/cond_gif/Assert/Assert}}]]
         [[IteratorGetNext]]
0 successful operations.
0 derived errors ignored. [Op:__inference_train_function_22071]

Function call stack:
train_function -> train_function

What should I do?
